service/framework.go

```diff
@@ -1,294 +0,0 @@
-package service
```

Comment:
This is now gone. There is no framework in the service package anymore. The framework is now in `service/certconfig/framework.go`.